broadinstitute/single_cell_portal_core
# Add feature flag for cross dataset search

This enables easy development of cross dataset search, in parallel with existing code, by adding a feature flag.

To test:
1. Do migration: run `ruby rails_local_setup.rb`, then `source config/secrets/.source_env.bash`, then `rails db:migrate`
2. Load home page, verify tab says "Search Studies"
3. Set feature flag for yourself: go to https://localhost:3000/single_cell/admin#users, select your account, then set "cross_dataset_search" to "True".
4. Load home page, verify tab says "Search Studies (XDS feature flag)"

This satisfies SCP-3280.
